assets/src/dashboard/app/views/shared/storyGridView.js

```diff
@@ -137,7 +141,14 @@ const StoryGridView = ({
             <DetailRow>
               <CardTitle
                 title={story.title}
-                modifiedDate={story.modified.startOf('day').fromNow()}
+                author={users[story.author].name}
+                status={story.status}
+                // this is fine according to the i18n package: https://developer.wordpress.com/themes/i18n/
+                // eslint-disable-next-line @wordpress/i18n-no-variables
+                displayDate={`${__(
+                  getDisplayDateVerb(story.status),
+                  'web-stories'
+                )} ${getFormattedDisplayDate(story.modified)}`}
```

Comment:
Because this runs it through moment each render, let's actually calc this whole string in a `useMemo` that way it only changes on status/modified updates.

Reply:
Just saw this now.

Please note that the text passed to `__()` needs to be a valid string literal, not a function call like `getDisplayDateVerb(story.status)`.

Maybe use `__()` within `getDisplayDateVerb()`? What does it return?

Reply:
It returns a moment formatted date string, nothing else. Thanks for the eyes on the translation formatting!